OpenLaTeXStudio/EditorModule/src/latexstudio/editor/menu/SaveFile.java

```diff
@@ -43,7 +43,8 @@
     @Override
     public void actionPerformed(ActionEvent e) {                
         String content = etc.getEditorContent();
-        File file = FileChooserService.getSelectedFile("tex", "TeX files", DialogType.SAVE, true);
+        File file = etc.getCurrentFile();
+        if (file == null) file = FileChooserService.getSelectedFile("tex", "TeX files", DialogType.SAVE, true);

```

Comment:
We should not alter this behaviour until "Save as" option becomes available. This change blocks the ability for the user to save as a new file. We already have a ticket for "save as" option, so this should be done there.